ee_plugin/ui/forms/add_ee_image.py

```diff
@@ -0,0 +1,101 @@
+import json
+from typing import Callable, Optional
+
+import ee
+from qgis.PyQt import QtWidgets
+from qgis.core import QgsMessageLog, Qgis
+
+from ...Map import addLayer
+from ..widgets import build_form_group_box, build_vbox_dialog
+from ..utils import call_func_with_values
+
+
+def form(accepted: Optional[Callable] = None, **dialog_kwargs) -> QtWidgets.QDialog:
+    """Display a dialog to add a GEE dataset to the QGIS map."""
+    dialog = build_vbox_dialog(
+        windowTitle="Add Google Earth Engine Image",
+        widgets=[
+            build_form_group_box(
+                title="Dataset",
+                rows=[
+                    (
+                        QtWidgets.QLabel(
+                            text="Enter GEE Image Name (e.g., COPERNICUS/S2, USGS/SRTMGL1_003)",
```

Comment:
If we want to conform to the style of the previous form, we would separate the example from the label with a newline and mark the example as `code`. Additionally, we would write the label as what is to be inputted (ie "GEE Image Name") as opposed to an instruction ("_Enter_ GEE Image Name").

```suggestion
                            text=text="<br />".join(
                                [
                                    _("GEE Image Name"),
                                    "e.g. <code>COPERNICUS/S2, USGS/SRTMGL1_003</code>",
                                ]
                            ),
```

Feel free to push back on that